meetings/cpansec-minutes-2024-11-15.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: page
+toc: true
+meeting_time: 2024-11-15 16:00 UTC
+title: Minutes 2024-11-15
+---
+
+## Meeting details
+
+* 2024-11-15 16:08 UTC on #cpansec-discussion on Matrix
+
+## 15:30 UTC – Pre-meeting socializing
+
+*   Socializing & getting up to speed before the meeting starts properly
+*   Discuss organizing projects, swimlanes and issues (...)
+
+## 16:00 UTC – Meeting start
+
+### Welcome
+
+*   @thibaultduponchelle joined :)
+*   Meeting chair: @timlegge
+*   Meeting secretary: @sjn & @tux
+
+### Attendees, absents & regrets
+
+*   Attendees
+    * @sjn, @stigtsp, @tux, @thibaultduponchelle, @timlegge
+*   Regrets
+    * @rrwo
+*   Partly Absent
+    * 
```

Comment:
It is good to have this in the template, but I guess it should drop in the accepted minutes once approved/stored/recorded